pyvista/core/utilities/reader.py

```diff
@@ -2434,6 +2435,11 @@ class XdmfReader(BaseReader, PointCellDataSelection):
     _vtk_module_name = "vtkIOXdmf2"
     _vtk_class_name = "vtkXdmfReader"
 
+    def __init__(self, path):
+        super().__init__(path)
+        self._active_time_value = 0.0
```

Comment:
There is no guarantee in most file formats that there will be a time point at 0.0.  I assume it is the same here.  We might have to set it after the information is read using `_set_defaults_post`.  To be backwards compatible we will need to know what the current behavior is, usually the first time point.

Reply:
Thanks. You are right that the first step is not zero. And I also understood that _set_defaults_post is the way to solve this problem. This abstract class is very well done!
The first step is active by default, so the latest changes should be fine.